src/index.js

```diff
@@ -1,11 +1,47 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore, compose } from 'redux';
 import './stylesheet/index.css';
 import App from './components/App';
+import booksReducer from './reducers/book';
+
+const randomID = () => Math.floor(Math.random() * 10);
+
+const initialState = {
+  books: [
+    {
+      ID: randomID(),
+      title: 'House of Sticks',
+      category: 'Biography',
+    },
+    {
+      ID: randomID(),
+      title: 'The Hidden Knife',
+      category: 'Kids',
+    },
+    {
+      ID: randomID(),
+      title: 'Survive The Night',
+      category: 'Horror',
```

Comment:
### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'House of Sticks', category: 'Biography'}